baselines/flanders/flanders/conf/base.yaml

```diff
@@ -0,0 +1,33 @@
+---
+# this is the config that will be loaded as default by main.py
+# Please follow the provided structure (this will ensuring all baseline follow
+# a similar configuration structure and hence be easy to customise)
+#defaults:
+#  - _self_
+#  - override hydra/hydra_logging: disabled
+#  - override hydra/job_logging: disabled
+
+hydra:
+  output_subdir: null
+  run:
+    dir: .
+
+dataset: mnist
+
+strategy:
+  _target_: flanders.strategy.Flanders
+  to_keep: 1  # number of benign local models to keep for the aggregation (FedAvg)
+  maxiter: 50 # number of iterations done by MAR
+
+server:
+  _target_: flanders.server.EnhancedServer
+  num_rounds: 50
+  pool_size: 10
+  warmup_rounds: 30
+  sampling: 500
+  history_dir: clients_params
+  magnitude: 100
+  threshold: 1e-05
+  attack_fn: gaussian
+  num_malicious: 0
+  omniscent: True
```

Comment:
```suggestion
  omniscent: True
 
 server_device: cpu
 
 client_resources:
   num_cpus: 10
   num_gpus: 0.0
  
```

Reply:
It's desirable to keep the `client_resources` as part of the config, so it's easier to modify from the command line. There is a comment related to this in `main.py`